test/fuzzers/frame_fuzzer_net.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include <iostream>
+#include <thread>
+
+#include <folly/init/Init.h>
+#include <folly/portability/GFlags.h>
+
+#include "rsocket/RSocket.h"
+#include "rsocket/transports/tcp/TcpConnectionAcceptor.h"
+#include "yarpl/Flowable.h"
+
+using namespace rsocket;
+using namespace yarpl::flowable;
+
+DEFINE_int32(port, 9898, "port to connect to");
+
+struct FrameFuzzerResponder : public rsocket::RSocketResponder {
```

Comment:
This doesn't seem to have anything regarding fuzzing here, it's just a server with a default responder.

I'd drop the mentions to fuzzing and rename things along the lines of `EmptyServer.cpp` or `DefaultServer.cpp`